app/services/ccmcc_evidence_check_rules.rb

```diff
@@ -26,6 +26,10 @@ def rule_applies?
     fee_range_applies?
   end
 
+  def office_id
+    @office_id ||= Office.find_by(entity_code: OFFICE_CODE).id
```

Comment:
How about we use the `.pluck()` method instead? 

```
Office.where(entity_code: OFFICE_CODE).pluck(:id)
```

I think it will be more efficient.